contracts/bprotocol/avatar/Avatar.sol

```diff
@@ -5,6 +5,10 @@ import { AbsCToken } from "./AbsCToken.sol";
 import { ICToken, ICEther, ICErc20 } from "../interfaces/CTokenInterfaces.sol";
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
+interface CompLike {
```

Comment:
In our code, we have all interfaces like `IXyz`. So we can rename it like `IComp`.

Reply:
I renamed and created a new file for it.